crates/sui-core/src/consensus_adapter.rs

```diff
@@ -283,11 +292,15 @@ impl ConsensusAdapter {
             ConsensusTransactionKind::UserTransaction(certificate) => {
                 let tx_digest = certificate.digest();
                 let position = self.submission_position(committee, tx_digest);
-                // DELAY_STEP is chosen as 1.5 * mean consensus delay
-                const DELAY_STEP: Duration = Duration::from_secs(7);
+                const DEFAULT_LATENCY: Duration = Duration::from_secs(5);
+                let latency = self.latency_observer.latency().unwrap_or(DEFAULT_LATENCY);
```

Comment:
Can we use DEFAULT_LATENCY as the min? This will still delay submissions from nodes that cannot submit to consensus. But it will make things a bit easier to reason about when there are both fast and slow nodes.

Reply:
Done